package.json

```diff
@@ -44,6 +44,7 @@
     "dotenv-cli": "^3.2.0",
     "eslint-plugin-prettier": "^3.4.0",
     "fast-check": "^1.26.0",
+    "italia-ts-commons": "^8.6.0",
```

Comment:
Wasn't this replaced by @pagopa/io-ts-commons?

Reply:
Good catch, fixed in https://github.com/pagopa/io-functions-admin/pull/165/commits/f2418a340bf44713202b930416cffba42cca3ce9